src/commands/ImportCommand.js

```diff
@@ -105,24 +117,39 @@ export default class ImportCommand extends Command {
     return ChildProcessUtilities.execSync(cmd, args, this.externalExecOpts);
   }
 
+  createPatchForCommit(sha) {
```

Comment:
Thanks for extracting this method :)

Reply:
No worries, it was my pleasure. Changes applied as requested